pkg/oc/cli/admin/policy/modify_roles.go

```diff
@@ -20,6 +20,8 @@ import (
 
 	userv1client "github.com/openshift/client-go/user/clientset/versioned/typed/user/v1"
 	authorizationutil "github.com/openshift/origin/pkg/authorization/util"
+
+	"k8s.io/apiserver/pkg/authentication/user"
```

Comment:
@deads2k I would expect this to fail with an import violation from the server to client.